doc/user/layouts/partials/create-source/connector/kinesis/with-options.html

```diff
@@ -1,5 +1,18 @@
-`access_key_id` | `text` | _(Required)_ A valid [access key ID](https://docs.aws.amazon.com/streams/latest/dev/controlling-access.html) to the Kinesis stream.
-`secret_access_key` | `text` | _(Required)_ A valid [secret access key](https://docs.aws.amazon.com/streams/latest/dev/controlling-access.html) to the Kinesis stream.
+`access_key_id` | `text` | A valid [access key ID](https://docs.aws.amazon.com/streams/latest/dev/controlling-access.html) to the Kinesis stream.
+`secret_access_key` | `text` | A valid [secret access key](https://docs.aws.amazon.com/streams/latest/dev/controlling-access.html) to the Kinesis stream.
+`token` | `text` | The session token associated with the credentials, if the credentials are temporary
 
 For details about the IAM account whose details you provide, see [Kinesis source
 details](#kinesis-source-details).
+
+If AWS credentials are not explicitly provided, Materialize wlil attempt to
```

Comment:
s/wlil/will

This could also be moved into active voice...

> If you do not explicitly provide AWS credentials, Materialize attempts to fetch...

Though not mandatory because I see the value in making "AWS credentials" the sentence's subject. Often times, tech writers mediate the conflict between active voice and a sentence's _true_ subject using headings (i.e. `#### AWS credentials`) but I digress.

Reply:
👍 done on both!